content-test/components/ActivityFeed.test.js

```diff
@@ -37,6 +37,12 @@ describe("ActivityFeedItem", function() {
     el = ReactDOM.findDOMNode(instance);
   });
 
+  it("should not throw if on missing props", () => {
```

Comment:
nit: not sure if you want "should not throw **if** missing props" or "should not throw **on** missing props".
Having both "**if on**" sounds a bit awkward (same w/ component-test/components/{SiteIcon,TopSites}.test.js below too)